CHANGE-NOTES.md

```diff
@@ -1,5 +1,9 @@
 # Changelog
 
+## v3.4.1
+
+- Improved: side-effecting actions (like Rebase, Pull, Discover etc.) are disabled while another side-effecting action is ongoing
```

Comment:
two questions here:

1. the ongoing rebase action != the ongoing git rebase, is it right?
2. one still can trigger some git machete side effecting action and the git4idea push "in parallel"? personally, I use `cmd+shift+K` (git4idea push) pretty often.

Reply:
> the ongoing rebase action != the ongoing git rebase, is it right?

Yes indeed, the idea (**not** set in stone ofc) is to just ban running Git Machete actions while another Git Machete action is running; other git4idea **not** included.

> one still can trigger some git machete side effecting action and the git4idea push "in parallel"? personally, I use cmd+shift+K (git4idea push) pretty often.

:yes:, huh indeed it'd be more accurate if we decided based on whether a push is ongoing in git4idea... but this will probably slow down `update()` massively ☹️ as compared to checking for a single atomic boolean value